Tomporvaz/add site button

tomporvaz opened the pull request:
# Pull Request 323

## Change Summary

Changed the Contribute button text to Add Site
Moved textTransform: none up to button sx

## Change Reason

Changed to avoid confusion about contributing information not money.

## Verification [Optional]
![Phlask_2023-07-18 21-10-56](https://github.com/phlask/phlask-map/assets/45155555/094d504a-799a-44ab-9a8c-60d6131c9984)

Related Issue: #323

tomporvaz committed:
Changed the Contribute button in Toolbar.js to say Add Site

tomporvaz committed:
Moved noWrap inline style to typography prop noWrap and moved textTransform to sx

RNR1 commented on src/components/Toolbar/Toolbar.js:
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Resources
-            </Typography>
+            <Typography fontSize={'small'}>Resources</Typography>
```
Curly braces are not necessary for the `fontSize` prop:
```suggestion
            <Typography fontSize="small">Resources</Typography>
```

RNR1 commented on src/components/Toolbar/Toolbar.js:
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Filter
-            </Typography>
+            <Typography fontSize={'small'}>Filter</Typography>
```
Same here:
```suggestion
            <Typography fontSize="small">Filter</Typography>
```

RNR1 commented on src/components/Toolbar/Toolbar.js:
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Search
-            </Typography>
+            <Typography fontSize={'small'}>Search</Typography>
```
Same here
```suggestion
            <Typography fontSize="small">Search</Typography>
```

RNR1 commented on src/components/Toolbar/Toolbar.js:
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
             <ContributeIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Contribute
+            <Typography noWrap fontSize={'small'}>
```
Same here:
```suggestion
            <Typography noWrap fontSize="small">
```

RNR1 reviewed (changes requested):
Looking great overall, just a few styling changes suggested here

tomporvaz committed:
Removed curly braces from fontSize = 'small'

tomporvaz committed:
Updated mobile ContributeButton to Add Site, and corrected more curly braces

tomporvaz commented:
@RNR1 I have removed the all of the curly brackets requested, and I also found more in the mobile code.  Searching for curly brackets, I realized that the toolbar is duplicated for mobile, so I updated the contribute button on mobile too.  I think this is complete, but let me know if there is anything else.

RNR1 reviewed (approved):
Thanks! Looking good
